tests/example/test_regular_examples.py

```diff
@@ -12,6 +12,7 @@
     observations_station_gaussian_model,
     observations_stations,
 )
+from tests import IS_LINUX
```

Comment:
`tests` is not always necessarily a real package. That's why pytest gave us fixtures, to overcome needing to import from specific locations, and many other reasons.